client/src/models/ConfigModel.ts

```diff
@@ -3,13 +3,20 @@
 
 import { Theme } from '@fluentui/theme';
 
-type PostCallSurveyType = 'msforms' | 'thirdparty';
-interface MSFormsSurveryOptions {
+export type PostCallSurveyType = 'msforms' | 'thirdparty';
+interface MSFormsSurveyOptions {
   surveyUrl: string;
 }
 interface ThirdPartySurveyOptions {
   surveyUrl: string;
 }
+
+export interface PostCallConfig {
+  survey?: {
```

Comment:
Should this be optional? `postCall` itself is optional, and we don't have any post-calls without survey, right?

Reply:
This was brought up during the tech doc discussion as well as later and the thought process explained was that we may have "other post call activities" and not just survey.